obal/data/tasks/brew_wait_and_download.yml

```diff
@@ -1,10 +1,13 @@
 ---
 # we use this to ensure that `wait` tagged tasks only run with `--tags
 # wait` is passed
-- command: /bin/true
+- name: 'Ensure wait tags run'
+  command: /bin/true
   register: normal_execution_check
+  when: True
```

Comment:
What does this add?

Reply:
There is a check to ensure that tasks are only run when there is a change. Which means either skipping the check here or adding this always true condition.